meteor/server/api/rundownLayouts.ts

```diff
@@ -88,6 +80,39 @@ PickerPOST.route('/shelfLayouts/upload/:showStyleBaseId', async (params, req: In
 	res.end(content)
 })
 
+/**
+ * Uploads the layout to the first Show Style Base that uses the given blueprint id
+ */
+PickerPOST.route(
+	'/shelfLayouts/uploadByBlueprintId/:showStyleBlueprintId',
```

Comment:
Since we expect a `showStyleBlueprintId` as the query parameter, should the path not reflect that?

Reply:
I renamed it